packages/wasm-dpp/test/unit/dataContract/DataContract.spec.js

```diff
@@ -324,14 +322,12 @@ describe('DataContract', () => {
     });
 
     it('should throw an error if document type is not found', () => {
-      let error;
```

Comment:
Same as above

Reply:
I see there is an `expect.fail` I didn't forget about!

Reply:
😄